test/fixtures/tab.ftl

```diff
@@ -0,0 +1,14 @@
+# Error (tab after =)
+key01 =	Value 01
+
+# Error (tab before =)
+key02	= Value 02
+
+# Error (tab is not a valid indent)
+key03 =
+	This line isn't properly indented.
+
+# Partial Error (tab is not a valid indent)
+key04 =
+    This line is indented by 4 spaces,
+	whereas this line by 1 tab.
```

Comment:
This exhibits the same behavior as when a broken attribute is encountered. The body of the message up until the syntax error parses fine. Essentially, it's the same situation as:

```properties
key04 =
    This line is indented by 4 spaces,
junk here
```